IdentityCore/src/util/NSString+MSIDExtensions.m

```diff
@@ -259,14 +259,21 @@ - (BOOL)msidIsEquivalentWithAnyAlias:(NSArray<NSString *> *)aliases
     {
         return NO;
     }
-
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Check if credential environment %@ is included in the list of valid environment aliases.",self);
+    for (NSString *alias in aliases)
+    {
+        MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, nil, @"environment alias found: %@",alias);
+    }
+    
     for (NSString *alias in aliases)
     {
         if ([self caseInsensitiveCompare:alias] == NSOrderedSame)
         {
+            MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"credential environment found in the list of valid environment aliases.");
             return YES;
         }
     }
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"credential environment not found in the list of valid environment aliases.");
```

Comment:
Same here, might be a bit too verbose for info level logs (all our authenticator logs are info level)

Reply:
Sounds good. I will change environment alias check logging to verbose then.